### ISSUE TITLE
custom request execute twice if all the rules success passed, with laravel v5.8.26

### ISSUE THREAD

**Issue by plhwin** [laravel/framework#28972]
- Laravel Version: v5.8.26
- PHP Version: 7.1.21
- Database Driver & Version: MySQL & 5.7.23

### Description:
A custom request class in Laravel will validate before entering a controller action, but all methods in the custom request will be executed twice if all the verification rules are passed. 

if some rules got error, executed only once.

### Steps To Reproduce:

Example Controller
```
<?php

namespace App\Http\Controllers\Api\V1;

use App\Http\Requests\Api\V1\TestRequest;
use Illuminate\Support\Facades\Log;


class AdminOrdersController extends \App\Http\Controllers\Controller
{

    public function cancel(TestRequest $request)
    {
        Log::info("test3:".time());
    }

}
```

Example Custom Request:
```
<?php

namespace App\Http\Requests\Api\V1;

use Illuminate\Foundation\Http\FormRequest;
use Illuminate\Support\Facades\Log;

class TestRequest extends FormRequest
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        Log::info("test1:".time());
        return true;
    }

    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return [
            'orderNo' => [
                'required', 'integer',
                'exists:orders,order_no',
                function ($attr, $value, $fail) {
                    Log::info("test2:".time());
                }
            ],
        ];
    }
}
```

for the example code above, if all the verification rules passed, logs like below:

```
[2019-06-27 13:01:58] local.INFO: test1:1561611718  
[2019-06-27 13:01:58] local.INFO: [SQL] select count(*) as aggregate from `orders` where `order_no` = ? ["5614450133994844"] 
[2019-06-27 13:01:58] local.INFO: test2:1561611718  
[2019-06-27 13:01:58] local.INFO: test1:1561611718  
[2019-06-27 13:01:58] local.INFO: [SQL] select count(*) as aggregate from `orders` where `order_no` = ? ["5614450133994844"] 
[2019-06-27 13:01:58] local.INFO: test2:1561611718  
[2019-06-27 13:01:58] local.INFO: test3:1561611718 
```

if some verification rules fails, logs like below:
```
[2019-06-27 13:04:10] local.INFO: test1:1561611850  
[2019-06-27 13:04:10] local.INFO: [SQL] select count(*) as aggregate from `orders` where `order_no` = ? ["5614450133994845"] 
[2019-06-27 13:04:10] local.INFO: test2:1561611850  
```

why executed twice if all the verification rules are passed?

**Comment by mfn** [comment]
Did you do anything special to "register" the request?

**Comment by plhwin** [comment]
> Did you do anything special to "register" the request?

nothing to do.

**Comment by plhwin** [comment]
found the reason, this is a bug caused by dingo/api, v2.2.3 is work. v2.2.4 and v2.3.0 are all meet this bug.

**Comment by Juhlinus** [comment]
@plhwin If it's caused by a 3rd-party-package then it does not apply here and should be closed. :)

**Comment by JamesKid** [comment]
I fork dingo/api project from 2.4.0 to my github repository , and i comment the twice call code , and i submit it in the composer "jameskid/dingo-api-fix-twice": "v1.0.0" , then , it's ok.
